routes/main.go

```diff
@@ -0,0 +1,15 @@
+package routes
+
+import "github.com/gin-gonic/gin"
+
+var router = gin.Default()
+
+func setupV1Routes(){
+	v1:= router.Group("v1/")
```

Comment:
which API format are we planning to go with "/api/v1" or "api.url"

Reply:
- api.<url>/V1/...